python/phylanx/ast/transformation.py

```diff
@@ -16,6 +16,9 @@
 
 et = phylanx.execution_tree
 
+# globals used during the compilation
+fglobals = {}
```

Comment:
What is the rational of making this 'global'?

Reply:
@hkaiser it is no longer global